keras_retinanet/bin/debug.py

```diff
@@ -177,7 +179,13 @@ def run(generator, args):
             image, image_scale = generator.resize_image(image)
             annotations[:, :4] *= image_scale
 
-        anchors = anchors_for_shape(image.shape)
+        # optionally load config parameters
+        anchor_params = None
```

Comment:
The config file parsing is written so to handle additional content in the future, hence it should not be uniquely linked to the anchor parameters. Please substitute these lines with:
```
   if args.config:
        args.config = read_config_file(args.config)

    anchor_params = None
    if args.config and 'anchor_parameters' in args.config:
        anchor_params = parse_anchor_parameters(args.config)
```